### PR TITLE
Adds new clothing for Security and the Captain.

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

This adds the Corporate Security Jacket, Corporate Uniform and the Captains Jacket, this also adds the seemingly unused Corporate Beret to the secdrobe

Full Corporate Security Outift:
https://gyazo.com/f4208bdb87577996e1ee944d36b87b59

Captain with his Jacket
https://gyazo.com/12418626e3dae6eaeec887c0612455d8

Stat-Wise they have a bit less armor but cover the arms too instead.

## Why It's Good For The Game

more cosmetics are always good.

## Changelog
:cl:
add: Added the corporate security uniform to the secdrobe, you can finally have your black security clothes.
add: Added a captains jacket to the captains locker.
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -43,6 +43,12 @@
 	item_state = "blueshift"
 	custom_premium_price = 600
 
+/obj/item/clothing/suit/armor/vest/corporate
+	name = "Corporate Jacket"
```
Comment (zeskorion) [inline code/modules/clothing/suits/armor.dm]: this is a jacket, but only has the area protection of a vest, and also has the armor of a vest? lower the armor and make it cover arms

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 2]
```diff
@@ -12,6 +12,7 @@
 	new /obj/item/storage/backpack/duffelbag/captain(src)
 	new /obj/item/clothing/neck/cloak/cap(src)
 	new /obj/item/clothing/neck/petcollar(src)
+	new	/obj/item/clothing/suit/armor/vest/capcarapce/jacket(src)
```
Comment (zeskorion) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: this gives captain the same issue HOS has, where he has multiple pieces of equally high-grade armor in one locker for cosmetic purposes.

File: code/modules/clothing/suits/armor.dm [hunk 3]
```diff
@@ -110,6 +116,12 @@
 	icon_state = "capformal"
 	item_state = "capspacesuit"
 
+/obj/item/clothing/suit/armor/vest/capcarapce/jacket
+	name = "captain's jacket"
```
Comment (zeskorion) [inline code/modules/clothing/suits/armor.dm:122]: same issue as sec has- vest protection but it's a jacket. lower armor, higher coverage.

File: code/modules/clothing/suits/armor.dm [hunk 4]
```diff
@@ -110,6 +116,12 @@
 	icon_state = "capformal"
 	item_state = "capspacesuit"
 
+/obj/item/clothing/suit/armor/vest/capcarapce/jacket
```
Comment (zeskorion) [inline code/modules/clothing/suits/armor.dm]:
```suggestion
/obj/item/clothing/suit/armor/vest/capcarapace/jacket
```

Comment (zeskorion) [reply code/modules/clothing/suits/armor.dm]: without this, this will be a normal secvest

**Review by zeskorion — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
@zeskorion alright, i gave the sec jacket 5 less armor overall and the captain jacket 10 less armor overall but made them both cover the arms too.

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm (hunk 2 above)
Comment (BeloneX) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: what do you propose then?

**Comment by qwertyquerty** [comment]
Not a fan of how these look

**Comment by ghost** [comment]
> Not a fan of how these look

i recorded this while having the game scaled to fit instead of pixel perfect 2 so its a bit off

File: code/modules/clothing/suits/armor.dm [hunk 5]
```diff
@@ -43,6 +43,14 @@
 	item_state = "blueshift"
 	custom_premium_price = 600
 
+/obj/item/clothing/suit/armor/vest/corporate
+	name = "Corporate Jacket"
```
Comment (Firefox13) [inline code/modules/clothing/suits/armor.dm]: Make it lowercase

**Review by zeskorion — APPROVED** [review]

**Pull request merged**